string      robot_name
int8        robot_id
int8[]      robot_route
float32     robot_v
float32     robot_p
float64[]   robot_coordinate
float32     robot_enter_time
bool        robot_enter_conflict
float32     robot_arrive_cp_time
float32     robot_exit_time